docs/sources/operators-guide/configuring/configure-object-storage-backend.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Configuring Grafana Mimir object storage backend"
+menuTitle: "Configuring object storage"
+description: "Learn how to configure Grafana Mimir to use different object storage backend implementations."
+weight: 70
+---
+
+# Configuring Grafana Mimir object storage backend
+
+Grafana Mimir can use different object storage services to persist blocks containing the metrics data, as well as recording rules and alertmanager state.
+The supported alternatives are:
+
+- [Amazon S3](https://aws.amazon.com/s3/) (and compatible implementations like [MinIO](https://min.io/))
+- [Google Cloud Storage](https://cloud.google.com/storage)
+- [Azure Blob Storage](https://azure.microsoft.com/es-es/services/storage/blobs/)
+- [Swift (OpenStack Object Storage)](https://wiki.openstack.org/wiki/Swift)
+
+Additionally, a file system emulated [`filesystem`]({{< relref "../configuring/reference-configuration-parameters/index.md#filesystem_storage_backend" >}}) object storage implementation can be also used for testing purposes (it's not recommended for production workloads).
+
+Ruler and alertmanager support a `local` implementation, which is similar to `filesystem` in the way that it uses the local file system, but it is a read-only data source and can be used to provision state into those components.
```

Comment:
Ruler also has a more detailed explanation of "local" that's worth to link: `docs/sources/operators-guide/architecture/components/ruler/index.md`

Reply:
Done in https://github.com/grafana/mimir/pull/2347/commits/c57665bef6823c2bcd25159bff2319723aba10d9